OrbitGl/CaptureWindow.cpp

```diff
@@ -637,6 +639,7 @@ void CaptureWindow::ResetHoverTimer() {
 
 //-----------------------------------------------------------------------------
 void CaptureWindow::Draw() {
+  auto start = std::chrono::high_resolution_clock::now();
```

Comment:
I think you want to use `std::chrono::steady_clock` here. `steady_clock` is guaranteed to be monotonic while `high_resolution_clock` is not.

From cppreference.com:
> The high_resolution_clock is not implemented consistently across different standard library implementations, and its > use should be avoided. It is often just an alias for std::chrono::steady_clock or std::chrono::system_clock, but which > one it is depends on the library or configuration. When it is a system_clock, it is not monotonic (e.g., the time can go > backwards). For example, for gcc's libstdc++ it is system_clock, for MSVC it is steady_clock, and for clang's libc++ > it depends on configuration.

> Generally one should just use std::chrono::steady_clock or std::chrono::system_clock directly instead of std::chrono::high_resolution_clock: use steady_clock for duration measurements, and system_clock for wall-clock time.
```

Reply:
Ah, thanks. Changed.